dashboard/pkg.jq

```diff
@@ -80,7 +80,8 @@ def makerow(r):
                       (?<m>\\d{2})   # month
                       (?<d>\\d{2})Z$ # day";
                     "\(.y)-\(.m)-\(.d)"; "ix")),
-        .value
+        .value,
+        "packagecloud"
```

Comment:
```suggestion
        "PackageCloud"
```

Reply:
These are how they show their names on their websites. I thought it would be more consistent to use their original names.